convert_sc_client_lookup.R

```diff
@@ -0,0 +1,271 @@
+  )
+
+
+# factor labels
+outfile <-
+  outfile %>%
+  mutate(
+    dementia = factor(dementia,
+      levels = c(0, 1), labels = c("No", "Yes")
+    ),
+    mental_health_problems = factor(mental_health_problems,
+      levels = c(0, 1), labels = c("No", "Yes")
+    ),
+    learning_disability = factor(learning_disability,
+      levels = c(0, 1),
+      labels = c("No", "Yes")
+    ),
+    physical_and_sensory_disability = factor(physical_and_sensory_disability,
+      levels = c(0, 1), labels = c("No", "Yes")
+    ),
+    drugs = factor(drugs,
+      levels = c(0, 1), labels = c("No", "Yes")
+    ),
+    alcohol = factor(alcohol,
+      levels = c(0, 1), labels = c("No", "Yes")
+    ),
+    palliative_care = factor(palliative_care,
+      levels = c(0, 1), labels = c("No", "Yes")
+    ),
+    carer = factor(carer,
+      levels = c(0, 1), labels = c("No", "Yes")
+    ),
+    elderly_frail = factor(elderly_frail,
+      levels = c(0, 1), labels = c("No", "Yes")
+    ),
+    neurological_condition = factor(neurological_condition,
+      levels = c(0, 1), labels = c("No", "Yes")
+    ),
+    autism = factor(autism,
+      levels = c(0, 1), labels = c("No", "Yes")
+    ),
+    other_vulnerable_groups = factor(other_vulnerable_groups,
+      levels = c(0, 1), labels = c("No", "Yes")
+    ),
+    living_alone = factor(living_alone,
+      levels = c(0, 1, 9), labels = c("No", "Yes", "Not Known")
+    ),
+    support_from_unpaid_carer = factor(support_from_unpaid_carer,
+      levels = c(0, 1, 9), labels = c("No", "Yes", "Not Known")
+    ),
+    social_worker = factor(social_worker,
+      levels = c(0, 1, 9), labels = c("No", "Yes", "Not Known")
+    ),
+    type_of_housing = factor(type_of_housing,
+      levels = c(1:6),
+      labels = c(
+        "Mainstream", "Supported", "Long Stay Care Home",
+        "Hospital or other medical establishment", "Other",
+        "Not Known"
+      )
+    ),
+    meals = factor(meals,
+      levels = c(0, 1, 9), labels = c("No", "Yes", "Not Known")
+    ),
+    day_care = factor(day_care,
+      levels = c(0, 1, 9), labels = c("No", "Yes", "Not Known")
+    )
+  )
+
+
+
+# rename variables
+outfile <-
+  outfile %>%
+  rename(
+    sc_dementia = "dementia",
+    sc_mental_health_problems = "mental_health_problems",
+    sc_learning_disability = "learning_disability",
+    sc_physical_and_sensory_disability = "physical_and_sensory_disability",
+    sc_drugs = "drugs",
+    sc_alcohol = "alcohol",
+    sc_palliative_care = "palliative_care",
+    sc_carer = "carer",
+    sc_elderly_frail = "elderly_frail",
+    sc_neurological_condition = "neurological_condition",
+    sc_autism = "autism",
+    sc_other_vulnerable_groups = "other_vulnerable_groups",
+    sc_living_alone = "living_alone",
+    sc_support_from_unpaid_carer = "support_from_unpaid_carer",
+    sc_social_worker = "social_worker",
+    sc_type_of_housing = "type_of_housing",
+    sc_meals = "meals",
+    sc_day_care = "day_care"
+  )
```

Comment:
See [rename_with](https://dplyr.tidyverse.org/reference/rename.html)

I think something like `rename_with(-c(sending_location, social_care_id), ~paste0("sc_", .x))` should work and would mean we don't have to update if we add variables.